src/components/UIElements/GifSeeker.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class='media'>
```

Comment:
This extra div means Vue's scoping will break any external CSS that expected the MediaEmbed to be a plain <img>.  Can work around it on a case by case basis with ::v-deep, but that's gonna be super easy to forget to test.